assets/js/util/i18n.js

```diff
@@ -83,13 +105,21 @@ const durationFormat = ( seconds, options = {} ) => {
 		} );
 	}
 
-	const formattedString = sprintf(
+	//  Use a fully internationalized approach in browsers that support it, with a fallback to XXh YYm ZZs.
+	if ( browserSupportsStyleUnit ) {
+		formattedString = sprintf(
 		/* translators: 1: formatted seconds, 2: formatted minutes, 3: formatted hours */
-		_x( '%3$s %2$s %1$s', 'duration of time: hh mm ss', 'google-site-kit' ),
-		seconds,
-		minutes,
-		hours,
-	);
+			_x( '%3$s %2$s %1$s', 'duration of time: hh mm ss', 'google-site-kit' ),
+			seconds,
+			minutes,
+			hours,
+		);
+	} else {
+		formattedString =
```

Comment:
Let's go with the consistent approach to localisation / translation rather than two different ways to do it, the only difference here is that we also add the units into the translatable string.:

```
formattedString = sprintf(
    /* translators: 1: formatted seconds, 2: formatted minutes, 3: formatted hours */
    _x( '%3$sh %2$sm %1$ss', 'duration of time with time units: XXh YYm ZZs', 'google-site-kit' ),
    seconds,
    minutes,
    hours,
);
```